### PR TITLE
Release preparations for v.1.1 (Python 3.4/3.5 support and Django 1.9 tests)

### PR DESCRIPTION
**Pull request by holgerd77**
Hi, sorry for this pull request, I don't want to take over your release management, I just can't wait to get this all up and running under `Python 3` :-) (I'am also eagerly waiting for `scrapy-splash` https://github.com/scrapinghub/scrapy-splash to catch up :-)).

If the pull request is ok it would be great if you can make a `PyPI` release as well!

Cheers
Holger

### CODE REVIEW

**Comment by kmike** [comment]
@holgerd77 thanks for the help! Could you please remove version bumps from the PR (it is easier/safer to do in master) and squash/rebase?

**Comment by holgerd77** [comment]
Hi @kmike, with "remove version bumps", do you just mean remove it from the `bumpversion.cfg` file (actually I didn't exactly know what this was for)? Or something else?

**Comment by holgerd77** [comment]
@kmike ok, I now removed the version change from `bumpversion.cfg`, and kept it in `setup.py`, hope that's ok now. I accidentally reverted your link changes in the README.rst and readded that with the last commit (ripping this out of the first commit would be too complicated). This is displayed incorrectly in the `Files changed` tab (at least for me, maybe it's also a caching issue), but if you look directly at the last commit this should be ok.

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='scrapy-djangoitem',
-    version='1.0.0',
+    version='1.1.0',
```
Comment (kmike) [inline setup.py]: Could you please also remove 1.1 version bump from setup.py?

File: README.rst [hunk 2]
```diff
@@ -170,3 +183,34 @@ Notice that we modify the ``sys.path`` variable instead the ``PYTHONPATH``
...
+  tox
+  
+...using the configuration in ``tox.ini``. The ``Python`` interpreters
+used have to be installed locally on the system.
+
+
+Changelog
+=========
+
+**v.1.1** (2016-02-17)
```
Comment (kmike) [inline README.rst]: I'm not sure we'll make a release today, could you please remove the date?

File: README.rst [hunk 3]
```diff
@@ -6,16 +6,16 @@ scrapy-djangoitem
    :target: https://pypi.python.org/pypi/scrapy-djangoitem
    :alt: PyPI Version
 
-.. image:: https://img.shields.io/travis/scrapy-plugins/scrapy-djangoitem/master.svg
+.. image:: https://img.shields.io/travis/scrapy/scrapy-djangoitem/master.svg
```
Comment (kmike) [inline README.rst]: It looks like you've reverted changes to `:target:`, but not to `.. image::`

**Pull request closed without merging**